spec/license_scout/dependency_manager/habitat_spec.rb

```diff
@@ -208,7 +208,7 @@
         expect(csc.version).to eq("12.17.49-20180503181308")
         expect(csc.license.records.first.id).to eql("Apache-2.0")
         expect(csc.license.records.first.source).to eql("https://bldr.habitat.sh/v1/depot/channels/chef/unstable/pkgs/chef-server-ctl/12.17.49/20180503181308")
-        expect(subject.fetched_urls["chef/chef-server-ctl"]).to eql("https://bldr.habitat.sh/v1/depot/channels/chef/unstable/pkgs/chef-server-ctl/12.17.49/20180503181308")
+        expect(subject.fetched_urls["chef/chef-server-ctl"]).to eql("https://bldr.habitat.sh/v1/depot/channels/chef/stable/pkgs/chef-server-ctl/12.17.49/20180503181308")
```

Comment:
Similarly here, the comment says: "fetching dependencies specified by a full ident from the unstable channel" which leads me to think that the intent was that we would be pulling from unstable here but maybe our test setup has gone wrong somehow.